src/Microsoft.AspNetCore.Razor.Language/ChildTagDescriptor.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    [DebuggerDisplay("{DisplayName,nq}")]
+    public abstract class ChildTagDescriptor : IEquatable<ChildTagDescriptor>
+    {
+        public string Name { get; protected set; }
+
+        public string DisplayName { get; protected set; }
```

Comment:
Added `DisplayName` with the vision that child tags may end up having different displays in the future.